pubspec.yaml

```diff
@@ -12,11 +12,17 @@ dependencies:
     sdk: flutter
   cookie_jar: ^3.0.1
   device_info_plus: ^3.2.2
-  flutter_web_auth: ^0.4.1
+  
   http: ^0.13.4
   package_info_plus: 1.4.2
   path_provider: ^2.0.9
   web_socket_channel: ^2.2.0
+  # flutter_web_auth: ^0.4.1
+  # Temporary fork of flutter_web_auth addressing issue:
+  # https://github.com/flutter/flutter/issues/103561#issuecomment-1126416045
+  flutter_web_auth:
+    git:
```

Comment:
I recommend keeping ```flutter_web_auth: ^0.4.1``` and adding the git reference separately in ```dependency_overrides:``` as this will add a warning/reminder every time ```pub get``` is run.